Changelog for v3.1.1 Update (#1040)
Rangi42/polishedcrystal: CHANGELOG.md

- All Magikarp Jump forms are now included.

_Evolution Methods:_
- Machoke, Graveler (both Plain and Alolan forms), Haunter, and Kadabra now evolve using the Linking Cord item, and no longer evolve by level up when they have a specific amount of EVs.
- Bonsley evolves by leveling up while knowing Rock Throw.
- Aipom evolves by leveling up while knowing Double Slap.
- Koffing evolves into Galarian Weezing by leveling up while holding a Charcoal.
- Primape evolves into Annihilape by leveling up while knowing Outrage.
- Scyther evolves into Kleavor by leveling up while holding a Hard Stone.
- Girafarig evolves into Farigiraf by leveling up while knowing Zen Headbutt.

> - Dunsparce evolves into Dundunsparce at level 32.

Reviewer: *Dun-dun-dun…* hi, Dudunsparce, there are several typo instances for it.

Author: These newfangled pokemon always trip me up. <.<

- Mime Jr. evolves into Plain Mr. Mime at level 30 and Galarian Mr. Mime at Ice Path. You can also use an Ice Stone for the latter.
- Hisuian Sneasel, Galarian Corsola, and Galarian Mr. Mime evolve the way they do in the mainline games.
- In addition to evolving by leveling up in Ilex Forest and Ice Path, respectively, Eevee can evolve into Leafeon using a Leaf Stone and Glaceon using an Ice Stone.
- Non-Alolan Pokemon evolve into their Alolan form evolutions by using an Odd Souvenir.

_Typing And Stats:_
- Farfetch'd was reverted to Normal/Flying with the inclusion of Galarian Farfetch'd.
- Girafarig and Ferigeraf are now Dark/Psychic.
- Galarian Corsola and Cursola are Ghost/Rock.
- Lugia is Water/Flying.

## Gameplay Features

_Initial Options:_
- You can now choose a third player character sprite option.
- The options menu pages were reorganized so that properties of the world/setting are initial options and properties of your equipment/interface are in-game options.
- No Real Time Clock (RTC) was turned into a initial option.
- Evolve in Battle added as an initial option (see next section).

> - New EV settings for play include choosing whether or not EVs exist, are 510 max for any Pokemon (like modern games), or All in which a Pokemon can gain 255 EVs in every stat. These settings apply both to the player and to NPCs.

Reviewer: The max of EVs for each stat is 252 as in Gen. VI onwards, not 255.

- Affection is included and can be turned on and off.
- New Game+ now preserves Catch/Oval/Shiny Charm (in addition to your money, mom's money, battle points, and any boxed Pokemon like before).
- Nuzlocke Mode has been removed along with some features that were added as a replacement. This is because the option caused players to believe that the game was balanced around the feature and often suggested changes based on this misperception.

_Moves and Battle:_
- Tons of move animation and sprite changes.

> - You can not evolve during battle just like in the anime.

Reviewer: Pretty sure you meant "You can now" 😂

Author: ....Yeah. Oops!

>
> — Satoru Iwata, "[Iwata Asks – Pokémon HeartGold Version & SoulSilver Version](https://www.nintendo.co.uk/Iwata-Asks/Iwata-Asks-Pokemon-HeartGold-Version-SoulSilver-Version/Iwata-Asks-Pokemon-HeartGold-Version-SoulSilver-Version/1-Just-Making-The-Last-Train/1-Just-Making-The-Last-Train-225842.html)"

***Note: The following changelog may contain spoilers.***

## v3.1.1

This is a hotfix release that removes one bug that caused the overworld graphics to glitch out badly when hatching an egg.

## v3.1.0

This is the newest stable version release which finished a number of branches that were under development and added them to v3.0.0, along with a few other small additions.

### Additions
- Bloodmoon Ursaluna, with its signature ability "Mind's Eye", is now available. It can be obtained by leveling up Ursaring in Murky Swamp. There are now 46 variant Pokémon for a total of 391 unique Pokémon.
- Added the much requested NPC to point the players in the direction of the Go-Goggles.
- The Candy Jar can now be obtained in the Goldenrod Underground. Exp Candies can be bought as well.

> - Support for the GB Printer was added so that players can print the Deploma.

Reviewer 2: ```suggestion
- Support for the GB Printer was added so that players can print the Diploma.
```

- The Galarian Bird event is completed. They are now accessible through the Pokemon Go trainers in the old Rocket Hideout in Celadon City.

### Fixes
- Updated the title version label to v3.1.0 and the copyright year to 2025.
- A graphical bug with the microphone so that it is below the player sprite.

## v3.0.0

Previously known as the "9bit" Branch of development, this version represents the bulk of the myriad of changes to v3.0.0-beta, which are covered in the following subsections.

### Pokemon
The total number of unique Pokémon has risen to 390 (including Red Gyarados, Spiky-Eared Pichu, and Armored Mewtwo). Excluding regional and alternate forms, there are 289 species required to complete the Pokédex. In general, if a Pokémon has some connection to Generations I or II, it is very likely present in the game, except for Paradox Pokémon and Mega Evolutions.

_Availability:_
- Reincluded post-3.0.0-beta: Spearow, Fearow, Aipom, Goldeen, Seaking, Hoppip, Skiploom, Jumpluff, Lickitung, Stantler, Shuckle, Cleffa, Igglybuff, Smoochum, Deliberd.

> - Added in v3.1.1: Ambipom, Lickilicky, Wyrdeer, Ferigeraf, Paldean Wooper, Clodsire, Annihilape, Hisuian Growlithe, Hisuian Arcanine, Galarian Corsola, Cursola, Dudunsparce, Bonsley, Hisuian Voltorb, Hisuian Electrode, Galarian Farfetch'd, Sirfetch'd, Happiny, Kleavor, Mantyke, Galarian Meowth, Perrserker, Mime Jr., Galarian Mr. Mime, Mr. Rime, Hisuian Qwilfish, Overqwil, Hisuian Sneasel, Sneasler, Paldean Tauros (Plain, Fire, and Water), Ursaluna (Plain), Wynaut, Hisuian Typhlosian, Galarian Articuno, Galarian Zapdos, and Galarian Moltres. (The three legendary birds are still unavailable.)

Reviewer 2: ```suggestion
- Added in v3.1.1: Ambipom, Lickilicky, Wyrdeer, Farigiraf, Paldean Wooper, Clodsire, Annihilape, Hisuian Growlithe, Hisuian Arcanine, Galarian Corsola, Cursola, Dudunsparce, Bonsley, Hisuian Voltorb, Hisuian Electrode, Galarian Farfetch'd, Sirfetch'd, Happiny, Kleavor, Mantyke, Galarian Meowth, Perrserker, Mime Jr., Galarian Mr. Mime, Mr. Rime, Hisuian Qwilfish, Overqwil, Hisuian Sneasel, Sneasler, Paldean Tauros (Plain, Fire, and Water), Ursaluna (Plain), Wynaut, Hisuian Typhlosian, Galarian Articuno, Galarian Zapdos, and Galarian Moltres. (The three legendary birds are still unavailable.)
```

- All Magikarp Jump forms are now included.

_Evolution Methods:_
- Machoke, Graveler (both Plain and Alolan forms), Haunter, and Kadabra now evolve using the Linking Cord item, and no longer evolve by level up when they have a specific amount of EVs.
- Bonsley evolves by leveling up while knowing Rock Throw.
- Aipom evolves by leveling up while knowing Double Slap.
- Koffing evolves into Galarian Weezing by leveling up while holding a Charcoal.

> - Primape evolves into Annihilape by leveling up while knowing Outrage.

Reviewer 2: ```suggestion
- Primeape evolves into Annihilape by leveling up while knowing Outrage.
```

- Girafarig evolves into Farigiraf by leveling up while knowing Zen Headbutt.
- Dunsparce evolves into Dudunsparce at level 32.
- Hisuian Qwilfish evolves into Overqwil at level 33.
- Ursaring evolves into Ursaluna by leveling up at the Sinjoh Ruins or the Mystri Stage. You can also use a Moon Stone. It evolves into Bloodmoon Ursaluna by leveling up in Murky Swamp.
- Magneton evolves into Magnezone by leveling up in Magnet Tunnel or Dim Cave. You can also use a Thunderstone.
- Stantler evolves into Wyrdeer by leveling up at Rugged Road, Snowtop Mountain, Sinjoh Ruins, or Mystri Stage.
- Mime Jr. evolves into Plain Mr. Mime at level 30 and Galarian Mr. Mime at Ice Path. You can also use an Ice Stone for the latter.
- Hisuian Sneasel, Galarian Corsola, and Galarian Mr. Mime evolve the way they do in the mainline games.
- In addition to evolving by leveling up in Ilex Forest and Ice Path, respectively, Eevee can evolve into Leafeon using a Leaf Stone and Glaceon using an Ice Stone.
- Non-Alolan Pokemon evolve into their Alolan form evolutions by using an Odd Souvenir.

_Typing And Stats:_
- Farfetch'd was reverted to Normal/Flying with the inclusion of Galarian Farfetch'd.

> - Girafarig and Ferigiraf are now Dark/Psychic.

Reviewer 2: ```suggestion
- Girafarig and Farigiraf are now Dark/Psychic.
```

- Galarian Corsola and Cursola are Ghost/Rock.
- Lugia is Water/Flying.

### Gameplay Features

_Initial Options:_
- You can now choose a third player character sprite option.
- The options menu pages were reorganized so that properties of the world/setting are initial options and properties of your equipment/interface are in-game options.

> - No Real Time Clock (RTC) was turned into a initial option.

Reviewer 2: ```suggestion
- No Real Time Clock (RTC) was turned into an initial option.
```

- One Odd Souvenir is accessible before the Johto Elite Four from Elm for finishing the Mystery Egg quest.
- Mirror Herb can now be used to transfer egg moves to an empty move slot using the daycare.
- Mint Leaf can be traded to the Tea Lady in Celadon apartments to change your Pokemon's nature.
- Removed the Dawn Stone for the Oval Stone.
- Exp share and power items available pre-E4.
- Oak gives the Shiny Charm now.
- Catching Charm can now be gotten from Prof. Oak.
- Sweet Honey added which attracts wild pokemon at the level of your party lead. Additionally an NPC was added who will sell it to you in the Goldenrod Flower shop.
- You can now buy PewterCrunch.
- Add Battle Factory items.
- Fourth Trainer Star is for beating Palmer or Thorton.
- Mint Leaf can now be bought with BP in the Battle Tower.
- The following items were replaced:
    * Ring Target -> Linking Cord
    * Absorb Bulb -> Clear Amulet
    * Cell Battery -> Mirror Herb
    * LuminousMoss -> PunchinGlove
    * Snowball -> Covert Cloak
    * UtilUmbrella -> Loaded Dice
- Added GBC Sound key item.

> - The Pokedex UI is overhauled, and is extensive in use and function. This includes a New Unown mode UI, and it play cries in the new Dex entry mode.

Reviewer 2: ```suggestion
- The Pokedex UI is overhauled, and is extensive in use and function. This includes a New Unown mode UI, and it plays cries in the new Dex entry mode.
```

- Sound effect for new Premier Ball.
- Added Go-Goggles key item. 

_Overworld And NPCs:_
- Field moves can be used without teaching TMs/HMs, so long as a member of the party can learn the move (as Headbutt and Teleport are not TMs/HMs, this does not apply). This includes Flash which can be used by pressing A in the dark, but excludes Dig which has no shortcut to use. Fly can be used through the Pokegear Map.
- Added several Hidden Grottos throughout the Johto region.
- Allow fishing while surfing with its own graphics.
- Player turns around after using a Pokecenter.
- Added Trainer Tip signs for some new features and some more Advanced Tips to Kanto.
- There are now several new routes based off of pokewalker locations or otherwise connecting proximinal routes that weren't connected before, such as Magnet Tunnel, Rugged Road, and Snowtop Mountain. Additionally, Stormy Beach, Cherrygrove Bay, Murky Swamp, and Violet Outskirts were all populated with wildmons.
- Special attention was given to Snowtop Mountain in that you can't get phone calls there and that there is a cool interactive cutscene there where you can view the scenery.

> - Violet Outskirts was also given a spooky event. Take the Sylph Scope there for an evert mon.

Reviewer 2: ```suggestion
- Violet Outskirts was also given a spooky event. Take the Sylph Scope there for an event mon.
```

- Removed Celadon Outskirts.
- Dynamic level scaling based on badge number was included for both wildmons and trainers in certain locations.
    For Wildmons: Hidden grottoes, all optional areas, and the Safari Zone.

>     For Trainers: Route Leaders, and alls trainers in the optional routes.

Reviewer 2: ```suggestion
    For Trainers: Route Leaders, and all trainers in the optional routes.
```

- Goldenrod Museum is now open for business.
- Plays a sound when the Rocket member takes a gate toll.
- Toned down poison screen effect for accessibility.
- Vulcandth added to the NPCs in the dev room.
- Added sand footprints. 
- Imported some text stuff from Prism.
- Changed the abbreviation of Speed to Spe instead of Spd as it was ambiguous.
- Remove roam mon anti-oscillation feature. (In Gen II, and ONLY in Gen II, if you return to the map you previously visited, roamers are forbidden from jumping there.)
- Weather effects, including rain (with lightning 25% of the time), sandstorm, and snow, now occur in the overworld on certain maps on certain days.

_Miscellaneous:_
- Feature Glaceon in the intro to reflect Rangi's profile picture.
- Analogue Pocket support for the RTC-off setting.
- Updated credits for sprite work (gmerc, Farore) and contributions.

### Fixes
Note: Some of these fixes may be intra-version fixes, and may not be all carried over from v3.0.0-beta.

- Fully paralyzed animation is faster now.
- Poison status outside of battle now interacts with Poison Heal properly, dealing no damage.
- Run Away prevents being trapped in unfaithful build.

> - Updated a few egg move pools (like giving one to weavile).

Reviewer 2: ```suggestion
- Updated a few egg move pools (like giving one to Weavile).
```

- Sleep Talk bugs fixed.
- Destiny Bond bug fixed.
- Pokerus fixes.
- Bug Catching contest fixes.
- Switch mode bug fixes.
- Lots of tile fixes and updates.
- A number of givepoke fixes.
- Tons of graphical glitch fixes. And fixes to those fixes.
- Pokedex radio show bug fix.
- Future Sight bug fixes.
- Analytic bug fixes.
- Link Battle bug fixes. Including allowing to do it in the first place.
- Sound bug fixes for cries and background music on transitions.
- Scaled EXP rounding changes.
- Register regular Pikachu in dex if you remove Surf/Fly from the special Pikachu forms.
- Several typo fixes.
- Several NPC trainer party error fixes.
- Updated Johto and Kanto Pokedex area maps.
- Added Orange Islands Pokedex area map.
- Headbutt and Rock Smash locators for dex.
- Wonder Trade bug fixes, including the post-E4 event.
- Learnset updates.
- Dex tracks cosmic forms separately instead of only regional forms.
- Updated some Battle Tower Movesets.
- Updated many mons movepools, learnsets, and tm compatibility to include or exclude moves they should or shouldn't learn in line with current generations.
- Balanced some wildmon levels.
- Updated several trainer parties.

> - Fixed Ssveral Pokégear phone bugs.

Reviewer 2: ```suggestion
- Fixed Several Pokégear phone bugs.
```

- Removed some headbutt trees from Goldenrod City.
- Safeguard bugs.
- Respawn in New Bark post-Elite 4.
- Allow losing to Elm.
- Several typo fixes.
- Infinite loop with empty TM/HM list.
- Lower fish and tree item probabilities.
- Party member gender icons on the nickname screen.
- New game clock.
- Move maniac text bug.
- Updated mons to have current gen stats.
- Mom randomly calling player bug.
- Overworld ability effect bugs.
- Bad egg bug given from NPCs.
- Oak tells you to meet him to unlock Mt. Silver.
- Items couldn't be used during bug catching contest.
- Power boosting and crit-boosting item fixes, as well as Metal Powder, Quick Powder, and Eviolite.
- Damage calculation bugs.
- Storage system wasn't working properly.
- Updated exp yield to scale like SwSh.
- Ball used when owning exactly 255 species crash.
- Several move description fixes.
- Several breeding bugs including pokeballs not inheriting correctly.
- Odd eggs hatch in pokeballs.
- Egg hatching uses shiny palette.
- Several map tweaks.
- Many mini-graphic tweaks and fixes.
- Text alignment fixes.
- Gave the sailboat text in Goldenrod Harbor.
- Swimmers in the Battle Tower now have walking sprites.

> - Many graphic bugs fixes.

Reviewer 2: ```suggestion
- Many graphic bug fixes.
```

- Revise initial option descriptions.
- Make more items available.
- Show icon for Park Balls.
- Colored gender symbols in nickname screen.
- Vitimin bugs.
- Allow player to player trading before the first gym.
- Pokedex graphics bugs and fixes.
- Nickname menus after contest prize in bug catching contest and prizing.
- Fix allowing fly in multiple areas.
- Fixed cursed body text bug.
- Changed Sludge Bomb to special and Gunk Shot to physical.
- Prof. Elm battle scripting.
- Reset mom's money on new game.
- Vitamins and Wings now account for the (optional) 510 EV limit.
- Fix player EVs before a battle begins (stat recalculation before battle).
- Screen no longer flashes when a mon escapes from a Park Ball.
- Shellder can learn Bite and be fished in Slowpoke Well (with Super Rod).
- Battle Factory EVs follow EV settings.
- Link Battle Record bug.
- In-Battle Pickup permanently steals wildmons items.
- Made several events more like HGSS, such as Silver dropping their trainer card.
- Pokegear radio bug.
- Fixed fishing for different forms.
- Journal total count and entry bug fixes.
- Buena's Password now operates in the evening.
- Fixed party naming screen issues.

> - Marlon palatte fly glitch.

Reviewer 2: ```suggestion
- Marlon palette fly glitch.
```

- Sylveon breeding bug. No more eldritch Eevees.
- Tea/Hyper Training factors in EVs.
- Knock Off against Substitute interaction fix.
- Infinite LuckNumberShow wins bug.
- Fixed Sheer Force affecting Thief.
- Fix Tyrogue evolution in EV disable mode.
- Fixed surfing behind the lighthouse.
- Many many improvements to the Battle Engine. 
- Figy Berry fix.
- Fixed stat change bug.
- Stat lowering actions fixes.
- Multihit move fixes.
- All kinds of optimizations to have the game run cleaner.
- Fix "spinner" Trainers and made them more like RBY.
- Apricorn colors fixes.
- Magnet Train animation fixes.
- Fade in palettes and flash darkness fixes.
- Several rounds of noise fixes.
- Fixed labels and terminology to be more like the official games.
- Fixed Violet gym guy throwing you across the street bug.
- Battle Factory's closed doors fix.
- Critical hit chance calculations fix.
- Neutralizing Gas and Intimidate interactions.
- Lake of Rage softlock.

> - Regional Varients can now be obtained through Wonder Trade.

Reviewer 2: ```suggestion
- Regional Variants can now be obtained through Wonder Trade.
```

> - Can no longer wondertrade Spikey Ear Pichu.

Reviewer 2: ```suggestion
- Can no longer Wonder Trade Spiky-Eared Pichu.
```

# Changelog

> What's the point of going through all the trouble of the debug process if you're going to go and fiddle with the game afterwards…?
>
> — Satoru Iwata, "[Iwata Asks – Pokémon HeartGold Version & SoulSilver Version](https://www.nintendo.co.uk/Iwata-Asks/Iwata-Asks-Pokemon-HeartGold-Version-SoulSilver-Version/Iwata-Asks-Pokemon-HeartGold-Version-SoulSilver-Version/1-Just-Making-The-Last-Train/1-Just-Making-The-Last-Train-225842.html)"

> Polished Crystal v3.1.1 brings with it a lot of changes. What started as a Dex expansion, through three years of development it has evolved into a strikingly different game from the v3.0.0-beta release with numerous added features from newer generations, a plethora of code optimizations, and many quality of life improvements. Ultimately, the game aims to feel like a modern Pokemon game with retro graphics. The rule of thumb is to assume that things work like modern games, and if it doesn't, it is either a bug or an oversight.

Reviewer 3: None of these paragraphs are necessary. Just keep the quote then get into the changelog. This sort of summarizing belongs in the README or FAQ.

What follows is a short description alongside a list of the changes made to several iterations of Polished Crystal from the last stable release (v3.0.0-beta). This is not an exhaustive list, but covers all of the major changes, and a majority of the minor ones. There are content spoilers.

Note: There are some changes listed from v2.2.0 to v3.0.0-beta as well. These are not in any way exhaustive, and likely don't list several major changes.

> ## v3.1.1

Reviewer 3: This is not a great way to do a changelog. It's better to explicitly list what's been changed. The way I'd do it is figure out what got changed v3.0.0 -> v3.1.0, then v3.1.0 -> v3.1.1. Everything else is v2.2.0 to v3.3.0.

Reviewer 3: Sorry. Completely misread this.

What follows is a short description alongside a list of the changes made to several iterations of Polished Crystal from the last stable release (v3.0.0-beta). This is not an exhaustive list, but covers all of the major changes, and a majority of the minor ones. There are content spoilers.

Note: There are some changes listed from v2.2.0 to v3.0.0-beta as well. These are not in any way exhaustive, and likely don't list several major changes.

## v3.1.1

This is a hotfix release that removes one bug that caused the overworld graphics to glitch out badly when hatching an egg.

## v3.1.0

This is the newest stable version release which finished a number of branches that were under development and added them to v3.0.0, along with a few other small additions.

### Additions

> - Bloodmoon Ursaluna, with its signature ability "Mind's Eye", is now available. It can be obtained by leveling up Ursaring in Murky Swamp. 332 Pokemon now complete the pokedex.

Reviewer 2: Need to verify the number of Pokemon. 

Last I counted it was:
```
291 Pokémon species, including some new evolutions, plus 56 cosmetic forms (e.g., Magikarp patterns, Pikachu Fly/Surf, Arbok patterns, Unown Forms) and 46 variants (Alolan, Galarian, Hisuian, misc.)—for a total of 393 unique Pokémon.
```

Reviewer 2: I was off by two... (Egg and -1)

```
289 Pokémon species, including some new evolutions, plus 56 cosmetic forms (e.g., Magikarp patterns, Pikachu Fly/Surf, Arbok patterns, Unown Forms) and 46 variants (Alolan, Galarian, Hisuian, misc.)—for a total of 391 unique Pokémon.
```

Author: Is that the double checked number? And is that the number you want listed? I listed the number of pokemon it takes to complete the pokedex as opposed to the exact number of technically unique mons, which itself could be double checked as well. (Incidentally, was the Oak check for dex completion for the diploma updated with the G-birds and BM-Ursa inclusions? I forget if regional forms are needed for Oak's dex completion.)

Reviewer 2: You need 289 caught pokemon to complete Oak's dex.. Which corresponds with the base species.  
  
You don't need the G-Birds, BM-Ursa, ect... you just need the base birds, ect.

- Added the much requested NPC to point the players in the direction of the Go-Goggles.
- The Candy Jar can now be obtained in the Goldenrod Underground. Exp Candies can be bought as well.
- Support for the GB Printer was added so that players can print the Deploma.
- The Galarian Bird event is completed. They are now accessible through the Pokemon Go trainers in the old Rocket Hideout in Celadon City.

> - Cherrygrove Tree in the bay now has text without the items used for the G-Bird event.

Reviewer 2: ```suggestion
```
This was fixed before the release, and wasn't a bug carried over from v3.0.0, so we don't need to talk about it; it just falls under the Galarian Bird event.

### Fixes
- Use the correct Swimmer sprite for the Go-Goggles NPC.

> - Several fixes to the amount of exp the Exp Candies give, as well as to their price display.

Reviewer 2: ```suggestion
```
Likewise we don't need to talk about this because it was fixed before release (and wasn't a bug carried over from v3.0.0)

> - GB Printer brightness modifications.

Reviewer 2: ```suggestion
```
Same with this one; don't need to talk about it.

- Updated the title version label to v3.1.0 and the copyright year to 2025.
- A graphical bug with the microphone so that it is below the player sprite.

## v3.0.0

Previously known as the "9bit" Branch of development, this version represents the bulk of the myriad of changes to v3.0.0-beta, which are covered in the following subsections.

### Pokemon

> The total number of unique pokemon has risen to 335 (including Red Gyarados, Spikey-Ear Pichu, and Armored Mewtwo). Excluding regional and alternate forms, there is a total of 289 unique species. 331 Pokemon are currently available for full dex completion. In general, if a Pokemon shares some relation to Generations 1 or 2, then it is very likely in the game, with the exception of Paradox Pokemon and Mega Evolutions.

Reviewer 2: Again lets double check these Pokemon numbers.

Reviewer 2: ```suggestion
The total number of unique pokemon has risen to 390 (including Red Gyarados, Spikey-Ear Pichu, and Armored Mewtwo). Excluding regional and alternate forms, there is a total of 289 unique species.
```

Just keep it simple.

Reviewer 2: ```suggestion
The total number of unique Pokémon has risen to 390 (including Red Gyarados, Spiky-Eared Pichu, and Armored Mewtwo). Excluding regional and alternate forms, there are 289 species required to complete the Pokédex. In general, if a Pokémon has some connection to Generations I or II, it is very likely present in the game, except for Paradox Pokémon and Mega Evolutions.
```

- Added a nurse's office to Celadon University.
- Added an Endure move tutor.
- Added Kimono Girls Mako and Mina.
- Nature Changing NPC with mint leaf tea.
- Added a bunch of pokemon journals.
- Several NPC interactions (haircuts and the friendship option for Shrine Maiden Rei) now exclusively give affection.
- More NPC hints about all kinds of features and items.
- Olivine Lighthouse shines at night.
- Goldenrod Museum is now open for business.
- Plays a sound when the Rocket member takes a gate toll.
- Toned down poison screen effect for accessibility.
- Vulcandth added to the NPCs in the dev room.
- Added sand footprints. 
- Imported some text stuff from Prism.
- Changed the abbreviation of Speed to Spe instead of Spd as it was ambiguous.
- Remove roam mon anti-oscillation feature. (In Gen II, and ONLY in Gen II, if you return to the map you previously visited, roamers are forbidden from jumping there.)
- Weather effects, including rain (with lightning 25% of the time), sandstorm, and snow, now occur in the overworld on certain maps on certain days.

_Miscellaneous:_
- Feature Glaceon in the intro to reflect Rangi's profile picture.

> - Analogue Pocket support for the non-RTC version.

Reviewer 2: There is no longer a `non-RTC` _version_. Maybe be more accurate to say RTC-off mode or something similar.

- Updated credits for sprite work (gmerc, Farore) and contributions.

> ### Fixes

Reviewer 2: Maybe mention that some of **_these_** fixes may be intra-version fixes, and may not be all carried over from v3.0.0-beta. I know it be too difficult to differentiate, so a simple disclaimer is fine.

- Ball used when owning exactly 255 species crash.
- Several move description fixes.
- Several breeding bugs including pokeballs not inheriting correctly.
- Odd eggs hatch in pokeballs.
- Egg hatching uses shiny palette.
- Several map tweaks.
- Many mini-graphic tweaks and fixes.
- Text alignment fixes.
- Gave the sailboat text in Goldenrod Harbor.
- Swimmers in the Battle Tower now have walking sprites.
- Many graphic bugs fixes.
- Fixed Trick bug so it can be used if one mon has an item.
- Fixed catch tutorial.
- Fix frontier brain battles no occurring bug.
- NPC trainer mon's cries fix.
- Damage hit sounds now working properly fixed.
- Fixed display bug for level up moves for stone evos.
- Move fixes, including Water Pulse, Roost, Hex, Disable, Pay Day, Destiny Bond (both function and the weird text bug), Destiny Bond a second time, Knock Off, Bug Bite, Psystrike (damage calc), Perish Song (should still affect the user if the opponent is immune), Avalanche not checking for if the opponent has hurt you, and Reversal so it can actually crit.
- Fix AI teleport usage.
- Rebalanced fishing item drop rate.
- Fix party icon colors when varying colors by DVs is off.

> - Ghost types can be affected by Wrap/etc. They can still flee/switch. (Oct 1st 2022)   <<< need help with this one

Reviewer: There was a bug where Ghost-type Pokémon were fully immune to being bound, when they're supposed to be just immune to switch/escape preventions. They can now properly become bound and take the residual damage while retaining the ability to switch out or run away regardless of being bound.

* Fix: Magnet Train tracks were missing tiles.
* Fix: Shiny icon is the star from Gen 3+.
* Fix: SPRITE_DRAGONITE uses Dragonite's side view.

> * Remove: Rebattling Breeders like in B2/W2.

Reviewer 3: Removals need a section header.

Author: I can do that. Would you like me to find any removals and do the same for the v3.0.0 section? I can think of one off the top of my head there as well. Seems a bit extra to have a whole section for one thing though, but if that's the call, that's what I'll do.

Reviewer 2: Just add this one for now.
